packages/lib-classifier/src/store/SubjectStore.spec.js

```diff
@@ -121,6 +123,43 @@ describe('Model > SubjectStore', function () {
     })
   })
 
+  describe('single image subjects', function () {
+    let subjects
+    let imageSubjects = Factory.buildList('subject', 10, { locations: [{ 'image/png': 'https://foo.bar/example.png' }] })
+
+    before(function () {
+      subjects = mockSubjectStore(imageSubjects)
+    })
+
+    it('should be valid subjects', function () {
+      const expectedSubject = SingleImageSubject.create(imageSubjects[0])
+      expect(subjects.active).to.deep.equal(expectedSubject)
+    })
+  })
+
+  describe('subject groups', function () {
+    let subjects
+    let imageSubjects = Factory.buildList('subject', 25, { locations: [{ 'image/png': 'https://foo.bar/example.png' }] })
+
+    before(function () {
+      const subjectGroups = []
+      for (let i = 0; i < 6; i++) {
+        const subjectGroup = {
+          id: `${i}`,
+          subjects: imageSubjects
+        }
+        subjectGroups.push(subjectGroup)
+      }
+      subjects = mockSubjectStore(subjectGroups)
+    })
+
+    it('should be valid subjects', function () {
+      expect(subjects.active).to.exist()
+      expect(subjects.active.id).to.equal('0')
+      expect(subjects.active.subjects).to.have.lengthOf(25)
```

Comment:
`subjects.active.subjects` I think could be confusing. What about `subjects.active.groupedSubjects` instead? We could also rename `subjects` to `subjectsQueue`

Reply:
The naming here will ultimately depend on what this attribute is called in Panoptes. I think this model is going to change once we know what the API response looks like.